.gitmodules

```
@@ -1,9 +1,12 @@
 [submodule "lib/Adafruit_BusIO"]
```

Can you test if this will work?
```suggestion
[submodule "components/Adafruit_BusIO"]
```